djongo/sql2mongo/query.py

```diff
@@ -123,7 +123,13 @@ def parse(self):
                 self.selected_columns = ColumnSelectConverter(self, statement)
 
             elif tok.match(tokens.Keyword, 'FROM'):
-                FromConverter(self, statement)
+                ## FIX: FROM(subquery)
+                if hasattr(statement.next_token, 'tokens') and \
+                        len(statement.next_token.tokens) > 1 and \
+                        statement.next_token[-1].value == 'subquery':
+                    self.nested_from_query = NestedFromQueryConverter(self, statement)
```

Comment:
Could simplify a bit:
tokens = getattr(statement.next_token, 'tokens', [])
if len(tokens) > 1 and tokens[0].value == 'subquery':